R/functions-xcmsSwath.R

```diff
@@ -0,0 +1,33 @@
+## function "align" EICs
+.alignEics <- function(x, y) {
```

Comment:
Sorry for being picky, but could you rename that to `.align_chromatogram` or `.alignChromatogram` instead? Then the user knows that he/she is aligning chromatograms.

Reply:
No problem.

Reply:
Changed to `.align_chromatogram`

Reply:
👍